developer/src/tike/project/Keyman.Developer.System.Project.Project.pas

```diff
@@ -4,13 +4,88 @@ interface
 
 uses
   System.Classes,
+  Winapi.Messages,
+
   Keyman.Developer.System.Project.ProjectFile;
 
 var
   FGlobalProject: TProject = nil;
   FGlobalProjectRefresh: TNotifyEvent = nil;
   FGlobalProjectRefreshCaption: TNotifyEvent = nil;
 
+const
+  WM_USER_ProjectUpdateDisplayState = WM_USER;
+
+function GlobalProjectStateWndHandle: THandle;
+
 implementation
 
+uses
+  System.SysUtils,
+  Winapi.Windows;
+
+type
+  TGlobalProjectStateWnd = class
```

Comment:
This class is moved from Keyman.Developer.System.Project.ProjectFile.pas, to keep it together with other globals. It is otherwise unmodified.